cppcache/include/geode/ResultCollector.hpp

```diff
@@ -20,10 +15,17 @@
  * limitations under the License.
  */
 
+#pragma once
+
+#ifndef GEODE_RESULTCOLLECTOR_H_
+#define GEODE_RESULTCOLLECTOR_H_
+
+#include <memory>
```

Comment:
For the moment global_types.h already includes <memory>

Reply:
I am not sure what mean here.

Reply:
global_types.h includes <memory>